Modules/zlibmodule.c

```diff
@@ -1434,16 +1455,39 @@ PyInit_zlib(void)
 #ifdef Z_TREES // 1.2.3.4, only for inflate
     PyModule_AddIntMacro(m, Z_TREES);
```

Comment:
Would it be possible to add error handling to PyModule_AddXXX() calls?

Reply:
@vstinner done.  It was trickier than I expected, if I just wrapped `Py_ModuleAddIntMacro` in my macro it didn't work.  I don't really understand why but wrapping `PyModule_AddIntConstant` works so I went with that.